src/components/Camera/AutoCapture.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+import * as React from 'react'
+import { h } from 'preact'
+import Visibility from 'visibilityjs'
```

Comment:
This library is nice to know about, as it could prove useful for tracking user actions. How often a user needs to tab away from CP would be a great metric for seeing how successful developments like docupedia have been at reducing overall 'non-CP' time a user has to be spent in order to complete a document task.

Reply:
Maybe this comment could be share on an internal forum instead since it is a bit out of scope for this public PR. ;)